### DISCUSSION TITLE
安装 WSL 报错：请启用虚拟机平台 Windows 功能并确保在 BIOS 中启用虚拟化

### DISCUSSION THREAD

**Discussion by Harry-Chen** [physics-data/faq#31] [Q&A] [answered]
如下图：

<img width="471" alt="4" src="https://user-images.githubusercontent.com/68137215/129466104-52d828ec-0be5-404c-9e2f-933691d61477.png">

**Comment by Harry-Chen** [comment, marked as answer]
请启用计算机的虚拟化功能，在 BIOS 配置中寻找类似以下名称的选项：

- VT-d / VT-x (Intel CPU)
- SVM Mode (AMD CPU)
- Virtualization (通用)

打开后重启，重新运行脚本。

如果找不到这些选项，请在此处报告你的计算机型号。

**Reply by yuchenguommm** [reply]
我应该是开了虚拟化的，如下
![623ff69686d63603ec910fc68df76b3](https://user-images.githubusercontent.com/88889094/129498326-ed4351e0-acad-405a-a71d-d63bb28c9b90.jpg)

**Comment by AuroraRyan** [comment]
可以WIN+R，输入optionalfeatures，打开windows功能界面，先检查一下Win虚拟机平台功能是否启用，有些电脑可能默认关闭
![微信图片_20210816151654](https://user-images.githubusercontent.com/88898895/129525665-ffaed928-1280-4b22-ac13-21e3f801f02b.png)

**Reply by jiegec** [reply]
或者在管理员权限的命令行里运行：`dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart`
